core/component.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/TheThingsNetwork/ttn/api"
 	pb_discovery "github.com/TheThingsNetwork/ttn/api/discovery"
+	pb_noc "github.com/TheThingsNetwork/ttn/api/noc"
```

Comment:
so do you mean getting rid of the alias and just importing it as `noc` ?
it seemed that the convention is to import everything from `/api/XXX` as `pb_XXX`